kale/predict/losses.py

```diff
@@ -274,3 +275,80 @@ def _moment_k(x: torch.Tensor, domain_labels: torch.Tensor, k_order=2):
             moment_sum += euclidean(x_k_order[i], x_k_order[j])
             n_pair += 1
     return moment_sum / n_pair
+
+
+class protonet_loss:
+    """
+    ProtoNet loss function.
+    This is a loss function for prototypical networks and N-way-K-shot problems. It computes the loss and
+    accuracy of the model by measuring the Euclidean distance between features of support samples and query samples.
```

Comment:
Did you explain anywhere what is support and what is query?

Reply:
Do we have to explain those fundamental concepts in the domain?